tests/block/009

```diff
@@ -0,0 +1,54 @@
+#!/bin/bash
+#
+# check page-cache coherency after BLKDISCARD
+# Regression test for patch "block: Invalidate cache on discard v2"-
+# https://lkml.org/lkml/2017/3/22/789
+# Page cache should be dropped after successful discard
+#
+# Copyright (C) 2017 Dmitry Monakhov <dmonakhov@openvz.org>
+#
+# This program is free software: you can redistribute it and/or modify
+# it under the terms of the GNU General Public License as published by
+# the Free Software Foundation, either version 3 of the License, or
+# (at your option) any later version.
+#
+# This program is distributed in the hope that it will be useful,
+# but WITHOUT ANY WARRANTY; without even the implied warranty of
+# MERCHANTABILITY or FITNESS FOR A PARTICULAR PURPOSE.  See the
+# GNU General Public License for more details.
+#
+# You should have received a copy of the GNU General Public License
+# along with this program.  If not, see <http://www.gnu.org/licenses/>.
+
+DESCRIPTION="check page-cache coherency after BLKDISCARD"
+
+. ./common/rc
+. ./common/scsi_debug
+
+requires() {
+	_have_scsi_debug && _have_program blkdiscard && _have_program xfs_io
```

Comment:
blkdiscard comes from util-linux which is required to be installed, so no need to have an explicit check for it.